### ISSUE TITLE
Windows `v5.3rc04.post3` Python module failure

### ISSUE THREAD

**Issue by tbirdso** [InsightSoftwareConsortium/ITKPythonPackage#205]
https://github.com/InsightSoftwareConsortium/ITKSplitComponents/actions/runs/3128603934/jobs/5076652004

See fix for similar issue on macos

https://github.com/InsightSoftwareConsortium/ITKPythonPackage/pull/204

**Comment by tbirdso** [comment]
Underlying issue is resolved in https://github.com/InsightSoftwareConsortium/ITKPythonPackage/pull/206

Patched Windows and Linux archives have been uploaded to https://github.com/InsightSoftwareConsortium/ITKPythonBuilds/releases/tag/v5.3rc04.post3 . Waiting for access to a clean MacOS machine in order to patch MacOS archives.

**Comment by SimonRit** [comment]
Thanks! I can see that it fixed the issue in ITKSplitComponents but it did not on RTK's builts, see https://github.com/SimonRit/RTK/actions/runs/3182160673/jobs/5188134114. I did check that SimonRit/RTK@c568b79445823fd4af3ab396b8283ed83edfccbe fixes `build-test-cxx (ubuntu-20.04)`. It seems to be an RTK specific issue if ITKSplitComponents does not have this issue.
cc @LucasGandel

**Comment by tbirdso** [comment]
Hi @SimonRit , this may indeed be an RTK-specific issue. Looking back through the logs you provided, the first error in at least the Windows build is this:
```
-- RTK: Creating itkPyBufferRTK submodule.
CMake Error at wrapping/itkPyBufferRTK.wrap:4 (file):
  file failed to open for reading (Invalid argument):

    C:/P/IPP/ITK-source/ITK/Modules/Bridge/NumPy/include;C:/Python38-x64/include/../wrapping/PyBuffer.i.init
Call Stack (most recent call first):
  C:/P/IPP/ITK-source/ITK/Wrapping/macro_files/itk_auto_load_submodules.cmake:302 (include)
  wrapping/CMakeLists.txt:24 (itk_auto_load_submodules)
```
When we look at [`itkPyBufferRTK.wrap`](https://github.com/SimonRit/RTK/blob/master/wrapping/itkPyBufferRTK.wrap) we find that it is trying to read a file at `ITKBridgeNumpy_WRAP_DIR`:
```
# Find PyBuffer swig files from ITKBridgeNumPy sources
set(ITKBridgeNumpy_WRAP_DIR "${ITKBridgeNumPy_INCLUDE_DIRS}/../wrapping")

file(READ "${ITKBridgeNumpy_WRAP_DIR}/PyBuffer.i.init"
    PyBufferInit)
```

From these two snippets it seems that the value of  `ITKBridgeNumPy_INCLUDE_DIRS` was previously a single directory but is now a semicolon-delimited list of two directories `C:/P/IPP/ITK-source/ITK/Modules/Bridge/NumPy/include` and `C:/Python38-x64/include`. Updating `itkPyBufferRTK.wrap` to only use the first list element in setting `ITKBridgeNumpy_WRAP_DIR` may be an adequate workaround. I have confirmed that `ITK/Modules/Bridge/NumPy/wrapping/PyBuffer.i.init` still exists in the current ITK source.

At present I do not know why the value of `ITKBridgeNumPy_INCLUDE_DIRS` has changed to now include Python include dirs. That may be a question for @thewtex .

**Comment by tbirdso** [comment]
Patched v5.3rc04.post3 archives have been uploaded at https://github.com/InsightSoftwareConsortium/ITKPythonBuilds/releases/tag/v5.3rc04.post3. Re-running ITKSplitComponents CI to verify that the module build issue is resolved across platforms. https://github.com/InsightSoftwareConsortium/ITKSplitComponents/pull/62

**Comment by tbirdso** [comment]
Closing as CI now passes in https://github.com/InsightSoftwareConsortium/ITKSplitComponents/pull/62 with the patched archives.
